LanguageFeatures/Patterns/shared_case_scope_A02_t02.dart

```diff
@@ -53,7 +53,6 @@ String test1(num x) {
 String test2(num x) {
   switch (x) {
     case var a as int when a > 0:
-    case int a:
     case int a when a is Never:
```

Comment:
Ah, now I understand, it's the case on line 56 (`... Never:`) which isn't reachable when the `case int a:` is included. So we're fixing the test by removing code that has no diagnostics, and keeping the code that used to cause a warning. Tricky! ;-)